### DISCUSSION TITLE
Python 3.11 Support

### DISCUSSION THREAD

**Discussion by jaytaytay** [kedro-org/kedro#2048] [Idea]
Hello,

I see that Kedro v0.18.0 (1 Apr 2022) introduced support for python 3.9 (released 5 Oct 2020) and 3.10 (4 Oct 2021).

I was wondering when we can expect Kedro to support Python 3.11 (24 Oct 2022)?

Thanks,
Liam

**Comment by astrojuanlu** [comment]
Hello @jaytaytay, sorry for the long delay. We saw your question but never replied here. We're tracking Python 3.11 support here: https://github.com/kedro-org/kedro/issues/2270 and https://github.com/kedro-org/kedro-plugins/issues/154 please upvote that issue and subscribe there for updates. Thanks for your interest!
